Source/Research.h

```diff
@@ -5,27 +5,27 @@
 
 enum LabType
 {
-	ltBasic			= 1,
-	ltStandard		= 2,
-	ltAdvanced		= 3
+	ltBasic	= 1,
+	ltStandard = 2,
+	ltAdvanced = 3
 };
```

Comment:
This is kind of a separate issue, though I wanted to mention it here. We should probably use `enum class` in place of `enum`. As `enum class` has mandatory scoping (`LabType::ltBasic`), we should also remove the `lt` prefix on each enum value name.